core/server/master/src/main/java/alluxio/master/meta/RetryHandlingMetaMasterMasterClient.java

```diff
@@ -82,6 +83,17 @@ public long getId(final Address address) throws IOException {
         .getMasterId(), LOG, "GetId", "address=%s", address);
   }
 
+  /**
+   * Returns current cluster id.
+   *
+   * @return a master id
+   */
+  public String getClusterIdInternal() throws IOException {
```

Comment:
who will make this call?

Reply:
This function is the previous design, I will delete it.